Source/WebKit/InspectorGResources.cmake

```diff
@@ -8,11 +8,10 @@ macro(WEBKIT_BUILD_INSPECTOR_GRESOURCES _derived_sources_dir)
         VERBATIM
     )
 
-    add_custom_command(
-        OUTPUT ${_derived_sources_dir}/InspectorGResourceBundle.c ${_derived_sources_dir}/InspectorGResourceBundle.deps
-        DEPENDS ${_derived_sources_dir}/InspectorGResourceBundle.xml
-        DEPFILE ${_derived_sources_dir}/InspectorGResourceBundle.deps
-        COMMAND glib-compile-resources --generate --sourcedir=${_derived_sources_dir}/InspectorResources/WebInspectorUI --target=${_derived_sources_dir}/InspectorGResourceBundle.c --dependency-file=${_derived_sources_dir}/InspectorGResourceBundle.deps ${_derived_sources_dir}/InspectorGResourceBundle.xml
-        VERBATIM
+    GLIB_COMPILE_RESOURCES(
+        OUTPUT        ${_derived_sources_dir}/InspectorGResourceBundle.c
+        DEPFILE       ${_derived_sources_dir}/InspectorGResourceBundle.deps
+        SOURCE_XML    ${_derived_sources_dir}/InspectorGResourceBundle.xml
+        RESOURCE_DIRS ${_derived_sources_dir}/InspectorResources/WebInspectorUI
```

Comment:
I'm somewhat surprised that the style checker did not bark at the spaces between the keywords and the arguments. Now that I know it won't, probably I will be doing the same myself in the future&mdash;this kind of alignment makes some of these long invocations easier to follow for me!